apps/tad/base/deployment-pgadmin.yaml

```diff
@@ -12,16 +12,19 @@ spec:
       automountServiceAccountToken: false
       securityContext:
         fsGroup: 5050
+        runAsGroup: 5050
+        runAsUser: 5050
       initContainers:
-        - name: init-clean-sessions
-          image: busybox
-          command: ['sh', '-c', 'rm -rf /var/lib/pgadmin/sessions']
```

Comment:
What was the purpose of this code? Clean old sessions? Won't we be accumulating sessions now?

Reply:
That was the purpose and it works. but the init container is only run on first startup. We then tried a sidecar, but that is also not run. 

@uittenbroekrobbert suggested changing the permissions. so we are now trying that one